overrides/config/ftbquests/quests/chapters/mailbox.snbt

```diff
@@ -0,0 +1,400 @@
+{
+	default_hide_dependency_lines: false
+	default_quest_shape: ""
+	filename: "mailbox"
+	group: "12BCDDCCCA65FD77"
+	icon: "mighty_mail:mangrove_mail_box"
+	id: "41A25067DC4A9F95"
+	images: [{
+		height: 10.0d
+		image: "advancementframes:block/board_back"
+		rotation: 0.0d
+		width: 10.0d
+		x: 1.0d
+		y: 3.0d
+	}]
+	order_index: 0
+	quest_links: [ ]
+	quests: [
+		{
+			can_repeat: true
+			dependencies: ["740DD4098893630E"]
+			description: ["{chapter.1.quest.60.description.1}"]
+			hide_dependency_lines: true
+			hide_until_deps_complete: true
+			hide_until_deps_visible: true
+			icon: {
+				Count: 1
+				id: "kubejs:elegant_letter"
+				tag: { }
+			}
+			id: "55C7DE6FA0F4E388"
+			invisible: true
+			optional: true
+			rewards: [
+				{
+					exclude_from_claim_all: true
+					icon: "create:orange_toolbox"
+					id: "1E4F523BAB1211F7"
+					table_id: 6664182306022005570L
+					title: "{chapter.1.quest.60.reward.1.title}"
+					type: "loot"
+				}
+				{
+					exclude_from_claim_all: true
+					icon: "quark:crate"
+					id: "6B1393250318C85B"
+					table_id: 7614141430405098414L
+					title: "{chapter.1.quest.60.reward.2.title}"
+					type: "loot"
+				}
+				{
+					auto: "invisible"
+					command: "/execute at @p run playsound minecraft:entity.experience_orb.pickup player @p ~ ~ ~ 180 0.1 0.5"
```

Comment:
Commands executed by Quests should not use `@p`, as FTBQ executes them from the server, and they have been observed to be inconsistent in who they target. use `{p}` instead, as FTBQ replaces this sequence with the name of the player claiming the reward.
```suggestion
					command: "/execute at {p} run playsound minecraft:entity.experience_orb.pickup player @p ~ ~ ~ 180 0.1 0.5"
```